docs/source/user-guide/sql/window_functions.md

```diff
@@ -330,7 +330,7 @@ row_number()
     FROM employees;
 ```
 
-````sql
```

Comment:
What is weird is that this documentation is supposed to be auto generated

https://github.com/apache/datafusion/blob/00132da33d51f272ddb19116121bed9a1420a0a5/datafusion/functions-window/src/row_number.rs#L56-L55

This looks like the problem might be that there are two ```sql` blocks there -- let me try and fix it another way...